publication-commons/build.gradle

```diff
@@ -17,7 +18,10 @@ dependencies {
     api group: 'com.amazonaws', name: 'aws-lambda-java-core', version: '1.2.0'
     api group: 'com.amazonaws', name: 'aws-java-sdk-lambda', version: awsSdkVersion
     api group: 'com.amazonaws', name: 'aws-java-sdk-dynamodb', version: awsSdkVersion
+    api group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: nvaCommonsVersion
```

Comment:
line 10 you have an nva-commons version with later version. Also, maybe better to use `project.ext` for this